### PR TITLE
fishPlugins.fzf-fish: 9.3 -> 9.7

### PR DESCRIPTION
**Pull request by natsukium**
https://github.com/PatrickF1/fzf.fish/releases/tag/v9.4
https://github.com/PatrickF1/fzf.fish/releases/tag/v9.5
https://github.com/PatrickF1/fzf.fish/releases/tag/v9.6
https://github.com/PatrickF1/fzf.fish/releases/tag/v9.7

###### Description of changes

fixes #194226

Because buildFishPlugin calls fishPlugins.fzf which was introduced by #220384, I added pkgs to inputs and inherit pkgs.fzf explicitly.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by pacien — APPROVED** [review]
I made a few remarks. Otherwise,

Diff looks good.
Quickly tested through wrapFish, works.

File: pkgs/shells/fish/plugins/fzf-fish.nix [hunk 1]
```diff
@@ -1,31 +1,35 @@
-{ lib, stdenv, buildFishPlugin, fetchFromGitHub, fd, fzf, util-linux, clownfish, fishtape_3 }:
-
+{ lib, stdenv, pkgs, buildFishPlugin, fetchFromGitHub, fd, util-linux, procps, clownfish, fishtape_3, }:
+let
+  # buildFishPlugin calls fishPlugins.fzf without inherit
+  inherit (pkgs) fzf;
```
[the remark this answers is not in the document]
Comment (natsukium) [reply pkgs/shells/fish/plugins/fzf-fish.nix]: It is clear and easy to understand. Thank you!

File: pkgs/shells/fish/plugins/fzf-fish.nix [hunk 2]
```diff
@@ -35,6 +39,6 @@ buildFishPlugin rec {
     description = "Augment your fish command line with fzf key bindings";
     homepage = "https://github.com/PatrickF1/fzf.fish";
     license = licenses.mit;
-    maintainers = with maintainers; [ pacien ];
+    maintainers = with maintainers; [ pacien natsukium ];
```
[the remark this answers is not in the document]
Comment (natsukium) [reply pkgs/shells/fish/plugins/fzf-fish.nix:42]: done.

**Review by pacien — APPROVED** [review]
Thanks!

**Pull request merged**